app/src/main/res/values/strings.xml

```diff
@@ -452,6 +452,7 @@
     <string name="resize_zoom">Zoom</string>
     <string name="caption_auto_generated">Auto-generated</string>
     <!-- Caption Settings -->
+    <string name="caption_file_name">%1$s-%2$s</string>
```

Comment:
As TobiGr said, make this non-translatable by moving it to `donottranslate.xml`

Reply:
Done